web_ui/packages/ui/src/divider/divider.module.scss

```diff
@@ -1,3 +1,3 @@
 .divider {
-    background-color: var(--spectrum-global-color-gray-50) !important;
+    background-color: var(--spectrum-global-color-gray-300) !important;
```

Comment:
This doesn't seem directly related to your changes, are you sure about this change?

Reply:
It’s directly related; the current color isn’t visible in the Project List. Here’s how it looks at the moment:
![Screenshot 2025-05-20 at 14 17 19](https://github.com/user-attachments/assets/01a268a4-9e10-42d5-9a02-4ebc2aa6b6c7)

This is with my changes. I think it went unnoticed before because we were importing `Divider` from a different sources
![Screenshot 2025-05-20 at 14 19 08](https://github.com/user-attachments/assets/2c39968f-97b3-4ee2-adfb-177cb7c328af)